pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -155,15 +163,19 @@ def __init__(
         save_top_k: Optional[int] = None,
         save_weights_only: bool = False,
         mode: str = "min",
-        period: int = 1,
+        every_n_epochs: int = 1,
+        every_n_batches: int = 0,
```

Comment:
Would None be a better default ?

Reply:
see https://github.com/PyTorchLightning/pytorch-lightning/pull/6146#discussion_r582001270 for previous discussion